specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-05-01/README.md

```diff
@@ -0,0 +1,21 @@
+# General Availability API specification for FrontDoor Azure Resource
```

Comment:
Readme.md file is used for SDK generation in this repo, is it possible to rename this file to avoid confusing sdk generation?